build.gradle

```diff
@@ -45,46 +46,54 @@ dokkaHtml.configure {
     }
 }
 
-publishing {
-    publications {
-        create("default", MavenPublication) {
-            from(components["java"])
+android {
+    compileSdkVersion 30
+}
```

Comment:
Presumably we should move the `android {...}` block from `/lib/build.gradle` here?